### PR TITLE
Changed order of EV battery addition

### LINKED ISSUE #54: Integration Battery
The battery on the integration screen defaults to the 12v battery where as this would be better showing the EV battery

image

### PR DESCRIPTION
**Pull request by cdnninja**
Changed order of EV battery to try make the EV battery take priority in the dashboard.  An attempt to resolve #54

### CODE REVIEW

File: custom_components/kia_uvo/sensor.py [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     vehicle: Vehicle = hass.data[DOMAIN][DATA_VEHICLE_INSTANCE]
+    INSTRUMENTS = []
+    
+    if vehicle.engine_type is VEHICLE_ENGINE_TYPE.EV or vehicle.engine_type is VEHICLE_ENGINE_TYPE.PHEV:
+        INSTRUMENTS.append(("evBatteryPercentage", "EV Battery", "vehicleStatus.evStatus.batteryStatus", PERCENTAGE, "mdi:car-electric", DEVICE_CLASS_BATTERY))
 
     INSTRUMENTS = [
```
[the remark this answers is not in the document]
Comment (cdnninja) [reply custom_components/kia_uvo/sensor.py]: I added an append to correct this.

**Comment by cdnninja** [comment]
I am not certain this will correct the issue.  Could you test?  Ideally I also want to modify this to only add items that have valid data.

**Comment by fuatakgun** [comment]
wow, this works perfectly, thanks for taking care of this, can we fix ordering like this and squash all commits into one?

```
    INSTRUMENTS = []

    if vehicle.engine_type is VEHICLE_ENGINE_TYPE.EV or vehicle.engine_type is VEHICLE_ENGINE_TYPE.PHEV:
        INSTRUMENTS.append(("evBatteryPercentage", "EV Battery", "vehicleStatus.evStatus.batteryStatus", PERCENTAGE, "mdi:car-electric", DEVICE_CLASS_BATTERY))
        INSTRUMENTS.append(("evDrivingDistance", "Range by EV", "vehicleStatus.evStatus.drvDistance.0.rangeByFuel.evModeRange.value", DYNAMIC_DISTANCE_UNIT, "mdi:road-variant", None))
        INSTRUMENTS.append(("totalDrivingDistance", "Range Total", "vehicleStatus.evStatus.drvDistance.0.rangeByFuel.totalAvailableRange.value", DYNAMIC_DISTANCE_UNIT, "mdi:road-variant", None))
    if vehicle.engine_type is VEHICLE_ENGINE_TYPE.PHEV:
        INSTRUMENTS.append(("fuelDrivingDistance", "Range by Fuel", "vehicleStatus.evStatus.drvDistance.0.rangeByFuel.gasModeRange.value", DYNAMIC_DISTANCE_UNIT, "mdi:road-variant", None))
    if vehicle.engine_type is VEHICLE_ENGINE_TYPE.IC:
        INSTRUMENTS.append(("fuelDrivingDistance", "Range by Fuel", "vehicleStatus.dte.value", DYNAMIC_DISTANCE_UNIT, "mdi:road-variant", None))

    INSTRUMENTS.append(("odometer", "Odometer", "odometer.value", DYNAMIC_DISTANCE_UNIT, "mdi:speedometer", None))
    INSTRUMENTS.append(("lastUpdated", "Last Update", "last_updated", "None", "mdi:update", DEVICE_CLASS_TIMESTAMP))
    INSTRUMENTS.append(("geocodedLocation", "Geocoded Location", "vehicleLocation.geocodedLocation.display_name", None, "mdi:map", None))
    INSTRUMENTS.append(("carBattery", "Car Battery", "vehicleStatus.battery.batSoc", PERCENTAGE, "mdi:car-battery", DEVICE_CLASS_BATTERY))
```

**Comment by cdnninja** [comment]
Done.  Let me know if that is what you were looking for.

**Review by fuatakgun — APPROVED** [review]
looks good to me, thanks again

**Pull request merged**